CHANGES.md

```diff
@@ -14,6 +14,7 @@
  * Bugfix: KrakenFutures, timestamp was not returned in book.
  * Bugfix: Phemex, websocket subscription error.
  * Bugfix: OKX, liquidations subscription was never called.
+ * Bugfix: OKX, use publicly available channel for book updates.
```

Comment:
I'd call this an "update" referring more to a change in the exchange. this isnt a bug

Reply:
You're right, I copy pasted and didn't pay much attention. Changed now.